### DISCUSSION TITLE
In multiplayer, you can drag humans but the server will refuse this request.

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14197] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When a human is stunned, the player can drag him, but you will auto let go immediately

In fact, the client shows that you can drag but you can't actually drag. You have to wait for at least one second to drag after the human to be stunned, otherwise the server will refuse your request.

There is no influence on the campaign, but PVP requires high operation skills. This problem occurs frequently and easily causes lag.

### Reproduction steps

Test in single player

https://github.com/FakeFishGames/Barotrauma/assets/73634685/69ed81fb-2e86-4d78-9b02-9c430b881a7c

Test in multiplayer

https://github.com/FakeFishGames/Barotrauma/assets/73634685/9cfbdf22-d4c1-4161-a3e4-720cd119af21

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by zhu-rengong** [comment]
btw, after the game was updated (v1.4.6.0->v1.5.8.0), a restriction was added: the enemy's inventory can only be accessed when the enemy's stun timer exceeds one second. This greatly reduces the playability and operability of PVP, because a master can snatch one or even two equipments from others in one second.

**Reply by ghost** [reply]
I don't agree, you shouldn't be stripping people by using microstuns.
